### ISSUE TITLE
FNX-5103 ⁃ [Bug] Tab preview sometimes blank & sometimes scrolled

### ISSUE THREAD

**Issue by Cheap-Skate** [mozilla-mobile/fenix#1567]
Latest build, should show tab preview top of page see #1420 #1462,

Some issues with www.bbc.co.uk/news
(1) sometimes preview is blank
scroll to top of www.bbc.co.uk/news
tap the tabs button to see the home screen tab preview - probably shows the page preview
reload the page and check the preview a few times. It's sometimes blank
sometimes don't need to reload, just repeatedly switch between home screen and actual page content.

(2) scroll offsets result in offset preview
Scroll the page to a finite scroll offset and reload the page at its offset position
check the page preview - it shows an offset snapshot

(3) preview snapshot sometimes does not show all page elements
scroll www.bbc.co.uk/news to the top
reload the page
compare the tab preview with the actual page content

* sometimes the BBC logo is missing from the preview
* it usually shows "menu" rather than "Home More  v"
I think the preview snapshot is taken before all the elements (JS?) are loaded eg the BBC logo

I see (1) and (2) on arstechnica.com as well, not sure about (3)

**Comment by cadeyrn** [comment]
(1) is #1529

**Comment by kbrosnan** [comment]
Please try to report issues as independent problems. Lumping several ones together makes resolving the bug difficult.

**Comment by vesta0** [comment]
@Cheap-Skate thank you for filing this issue. This is no longer relevant considering the recent re-design of tabs and sessions. Please check out the new tab design and collections once it lands in the next couple of weeks.
